Neos.Fusion/Classes/Service/HtmlAugmenter.php

```diff
@@ -69,7 +69,7 @@ protected function getHtmlRootElement($html)
         $domDocument = new \DOMDocument('1.0', 'UTF-8');
         // ignore parsing errors
         $useInternalErrorsBackup = libxml_use_internal_errors(true);
-        $domDocument->loadHTML($html);
+        $domDocument->loadHTML(mb_convert_encoding($html, 'HTML-ENTITIES', 'UTF-8'));
```

Comment:
We came across this bug the other day and “solved” it by working avoiding special characters in attributes using transliteration.
Ofc this should be fixed in the core, but I'm afraid using `mb_convert_encoding()` could have _other_ side effects though (one of which being a potentially severe performance impact).

Maybe the encoding should be a property of the augmenter so we could opt-in to utf-8 (and make it a default for the next major)?

Reply:
@bwaidelich we use utf-8 as default in many places so noone can assume this to work in other charsets. On the other hand we should think about the possible performance impact. 

I also succeeded to fix the behavior by prefixing the html with `loadHTML('<?xml encoding="UTF-8">' . $html);` The result is the same and has probably less performance impact. We could still check wether the html starts with `<?xml` and skip the prefix in this case.

Reply:
What about going for the  work around (`<?xml` or transliteration or possibly a flag in the augmenter) for 3.3.x and to change the default behavior with 5.0?

Reply:
@bwaidelich pushed an update that uses the xml-prologue solution.